tests/data_context/test_configuration_storage.py

```diff
@@ -71,142 +71,3 @@ def data_context_parameterized_expectation_suite_with_usage_statistics_enabled(
         ),
     )
     return gx.get_context(context_root_dir=context_path)
-
-
-def test_preserve_comments_in_yml_after_adding_datasource(
-    data_context_parameterized_expectation_suite_with_usage_statistics_enabled,
-):
-    # Skipping this test for now, because the order of the contents of the returned CommentedMap is inconsistent.  # noqa: E501
-    pytest.skip("KNOWN ISSUE")
```

Comment:
This seems unrelated to removing actions. Is that right? Are we deleting this because it is always skipped now? This is reasonable, just want to make sure I understand.

Reply:
Yup its been skipped for years so thought I'd just remove it entirely - I can bring it back if we don't want to muddy up the diff though